shared/AppInsightsCore/src/JavaScriptSDK.Interfaces/IProcessTelemetryContext.ts

```diff
@@ -41,7 +47,7 @@ export interface IProcessTelemetryContext {
      * @param field The config field name
      * @param defaultValue The default value to return if no defined config exists
      */
-    getConfig: (identifier: string, field: string, defaultValue?: number | string | boolean) => number | string | boolean;
+    getConfig: (identifier: string, field: string, defaultValue?: number | string | boolean | string[] | RegExp[] | Function) => number | string | boolean | string[] | RegExp[] | Function;
```

Comment:
Adding some more of the actual types so that users don't need to say "any" or "unknown"